torch/export/unflatten.py

```diff
@@ -788,6 +791,20 @@ def _outline_submodules(orig_graph: torch.fx.Graph, root_module: UnflattenedModu
     ).run_outer()
 
 
+def _reorder_submodules(parent: torch.nn.Module, fqn_order: Dict[str, int], prefix: str = ""):
+    children = []
+    for name, child in list(parent._modules.items()):
+        if child is None:
+            continue
+        fqn = prefix + name
+        _reorder_submodules(child, fqn_order, fqn + ".")
+        delattr(parent, name)
```

Comment:
Is this needed because `register_module` will error otherwise?

Reply:
yes